src/Uno.UI/Extensions/UIBarButtonItemExtensions.iOS.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Uno.Extensions;
+using UIKit;
+
+namespace Uno.UI.Extensions
+{
+	internal static class UIBarButtonItemExtensions
+	{
+		/// <summary>
+		/// 
```

Comment:
Missing the comments?

Reply:
Also, why did you choose to make it an extension instead of a method in the renderer ?

Reply:
@ajpinedam Good catch, I added the comment.

@jeromelaban No reason other than it made sense to me, I can move it to the renderer if it bothers you.

Reply:
That's fine, let's keep it there.